src/tiled/changelayer.cpp

```diff
@@ -138,7 +166,6 @@ void SetLayerOpacity::setOpacity(qreal opacity)
     emit mDocument->changed(LayerChangeEvent(mLayer, LayerChangeEvent::OpacityProperty));
 }
 
-
```

Comment:
There are two empty lines here because the definitions above and below and for different classes. Please keep it that way. :-)

Reply:
done